sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -80,20 +82,18 @@ trait MetadataMapSupport {
             .mkString("[", ", ", "]")
         case JInt(value) => value.toString
         case JDouble(value) => value.toString
+        case JLong(value) =>
+          if (timestampKeys.contains(key)) {
+            new Date(value).toString
```

Comment:
Could you use `Iso8601TimestampFormatter` instead of the legacy class `Date`.

Reply:
This is the behavior for DESCRIBE TABLE without AS JSON, see https://github.com/apache/spark/commit/36d23eff4b4c3a2b8fd301672e532132c96fdd68#diff-f2a04f920c41d18a7d387216f86405bfdc6fb09c44ebe1bb09312ba7dde55333L136

Here we just revert it back.